docs/docusaurus.config.js

```diff
@@ -16,56 +18,92 @@ const legacyVersion = {
   to: 'https://legacy-docs-v1.rasa.com',
 };
 
+// FIXME: when deploying this for real, change to '/docs/rasa/'
+const BASE_URL = '/docs/rasa/next/';
+const SITE_URL = 'https://rasa.com';
+
 module.exports = {
   title: 'Rasa Open Source Documentation',
   tagline: 'Rasa Open Source Documentation',
-  url: 'https://rasa.com',
-  // FIXME: when deploying this for real, change to '/docs/rasa/'
-  baseUrl: '/docs/rasa/next/',
+  url: SITE_URL,
+  baseUrl: BASE_URL,
   favicon: 'img/favicon.ico',
   organizationName: 'RasaHQ',
   projectName: 'rasa',
   themeConfig: {
+    colorMode: {
+      defaultMode: 'light',
+      disableSwitch: true,
```

Comment:
I think that a lot of people liked the dark mode - any reason why we do not keep it around?

Reply:
It's not within our design system, we don't have a dark-mode palette. It's already been warned about, but we intend to look at bringing it back.

Reply:
CC @eliseboyd ^^

Reply:
ok 👍